lib/licenses.nix

```diff
@@ -618,6 +618,12 @@ in mkLicense lset) ({
     fullName  = "University of Illinois/NCSA Open Source License";
   };
 
+  nosql-workbench = {
```

Comment:
If this is a license specifically for this one piece of software, I think maybe it would be better to just use licenses.unfree? I'd worry about licenses.nix getting huge if it includes every single-use proprietary license.

But I don't feel too strongly so if there's something that makes it convenient to have the license more explicit that's fine. :)

Reply:
Fair enough. I didn't know the best way to handle it. Yeah, it's a license specifically for this one piece of software.

Reply:
Makes sense that it's nice to provide the license.

One thing we could do would be to inline the license definition:

```
license = {
  fullName = "NoSQL Workbench License Agreement";
  description = "...";
  free = false;
};
```

I think this would be a new pattern, but might get us the best of both worlds?

Reply:
Oh, yeah, I love that! Avoids having to grep through the repo to find `licenses.nix`!